<launch>
  <arg name="input_topic_name" default="/sensing/lidar/top/pointcloud_raw_ex"/>
  <arg name="output_topic_name" default="/sensing/lidar/top/voxel_grid_downsample_filter/pointcloud"/>
  <arg name="input_frame" default="base_link"/>
  <arg name="output_frame" default="base_link"/>
  <arg name="voxel_grid_downsample_filter_param_file" default="$(find-pkg-share autoware_downsample_filters)/config/voxel_grid_downsample_filter_node.param.yaml"/>
  <node pkg="autoware_downsample_filters" exec="voxel_grid_downsample_filter_node" name="voxel_grid_downsample_filter_node">
    <param from="$(var voxel_grid_downsample_filter_param_file)"/>
    <remap from="input" to="$(var input_topic_name)"/>
    <remap from="output" to="$(var output_topic_name)"/>
    <param name="input_frame" value="$(var input_frame)"/>
    <param name="output_frame" value="$(var output_frame)"/>
  </node>
</launch>
